asmtest.json

```diff
@@ -5,7 +5,8 @@
     "test_path": "tests",
     "rendered_file": "test.bs",
     "before_each": [
-        "bin/bshift .asmtest/test.bs"
+        "bin/bshift --nasm .asmtest/test.bs",
+        "bin/bshift --basm .asmtest/test.bs"
```

Comment:
I might need to change asmtest to support this situation better. If I recall correctly, this will use nasm, then overwrite that with basm, then only test basm.